src/components/Friends/Friend/styles.scss

```diff
@@ -31,18 +31,22 @@
   flex-flow: row nowrap;
   align-items: center;
   justify-content: space-between;
-  & h2 {
+  & p {
     margin-bottom: 5px;
     color: grey;
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    font-weight: bold;
   }
 }
 .deleted-user-icon {
   font-size: 1.8em;
   color: gray;
   margin-right: 5px;
 }
+
 .friend-box-container {
-  width: 100%;
+  width: 75vw;
```

Comment:
This is just personal opinion, but I would like the search bar and friends boxes to always be same width. Now it's sometimes same width and sometimes different, depending on the screen size.

<img width="953" alt="Screenshot 2020-04-30 at 14 08 55" src="https://user-images.githubusercontent.com/23714794/80703854-37204c80-8aec-11ea-8766-a9a1ea1663ad.png">

Reply:
I agree, I did it this way because I noticed that on wider screens "Poistunut käyttäjä" and other user bars were of different length which looked even worse. This fixed that problem, and I couldn't find a neater solution in the time I had. Maybe should be fixed later..